app/src/main/java/com/framgia/englishconversation/data/model/TimelineModel.java

```diff
@@ -274,4 +293,10 @@ public String toString() {
                 + mCreatedUser
                 + '}';
     }
+
+    @IntDef({NOMARL, EDITOR_CHOISE})
+    public @interface Flag {
+        int NOMARL = 0;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.